examples/file-feed/complete/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "file-feed",
+  "version": "0.0.0",
+  "bin": "src/ff-cli/bin.js",
```

Comment:
Good! I was thinking we could yargsify the CLI tool so that we can run 
`ff-cli daemon hello-world` and `ff-cli add hello-world file.txt` instead of having two separate programs. This would make the code sharing between the two also a lot easier. Unless you're against yargsifying it, I'll go ahead and do that today.